frontend/src/components/admin/generalConfig/common/ConfigMenuDisplay.js

```diff
@@ -196,13 +196,38 @@ function ConfigMenuDisplay(props) {
                 <Section>
                   <Form onSubmit={handleModify}>
                     <Column lg={16} md={8} sm={4}>
-                      <Button disabled={modifyButton} type="submit">
+                      <Button
+                        disabled={modifyButton}
+                        type="submit"
+                        style={{
```

Comment:
@Bahati308 , we would better use carbon grid  for adjusting size and layout.
And also , for a duplicated styling , you would use a separate css class.
BUt also what you have done doesnt look like it improves responsiveness . this only affects a given screen size